### PR TITLE
Improve confusing comments

### PR DESCRIPTION
**Pull request by frederikrothenberger**
This PR improves confusing comments surfaced in #1926.

### CODE REVIEW

File: src/frontend/src/test-e2e/flows.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ export const FLOWS = {
     await welcomeView.login();
     await welcomeView.typeUserNumber(userNumber);
     await browser.$("button[data-action='continue']").click();
-    // NOTE: handle recovery nag because there is no recovery phrase
+    // This flows assumes no recovery phrase, so we handle recovery nag here
```
Comment (lmuntaner) [inline src/frontend/src/test-e2e/flows.ts:96]: Is it a typo `"this flows"`? Shouldn't it be singular?

Can you change the verb "handle" for "skip"? I found the "handle" part to be vague and confusing.

**Review by lmuntaner — COMMENTED** [review]
Thanks!! Just a minor comment.

File: src/frontend/src/test-e2e/flows.ts (hunk 1 above)
Comment (frederikrothenberger) [reply src/frontend/src/test-e2e/flows.ts]: Sure, I was a bit too quick with this one. 🙃

**Review by lmuntaner — APPROVED** [review]
Thanks!

**Pull request merged**